orders-war/gulpfile.js

```diff
@@ -3,28 +3,26 @@
 var gulp = require('gulp');
 var path = require('path');
 var replace = require('gulp-replace');
-var del = require('del');
+var filter = require('gulp-filter');
 
-gulp.task('web-template-release-Clean', function () {
-    return del(['./target/generated-sources/src/main/resources/templates/web-template-release/**'], {force:true});
-});
```

Comment:
Should this still clear the old files before updating with the new?
I suppose it's not necessary, but unused files could remain

Reply:
Never mind, I see that the clean has moved to package.json